sample/src/main/java/kin/sdk/core/sample/CreateAccountActivity.java

```diff
@@ -51,7 +51,7 @@ Intent getBackIntent() {
 
     @Override
     int getActionBarTitleRes() {
-        return R.string.create_account;
+        return R.string.create_wallet;
```

Comment:
So maybe we can rename the activity name to CreateWalletActivity?

Reply:
👍